src/org/labkey/test/components/react/BaseReactSelect.java

```diff
@@ -36,6 +36,10 @@ public abstract class BaseReactSelect<T extends BaseReactSelect<T>> extends WebD
     private static final String LOADING_TEXT = "loading...";
     protected static final String SELECTOR_CLASS = "select-input-container";
 
+    protected int READY_TIMEOUT = WAIT_FOR_JAVASCRIPT;
+    protected int OPEN_TIMEOUT = 4_000;
```

Comment:
[Java convention] All-caps generally indicates that an identifier is `final`
```suggestion
    protected int readyTimeout = WAIT_FOR_JAVASCRIPT;
    protected int openTimeout = 4_000;
```